Makefile

```diff
@@ -74,7 +74,7 @@ clean:
 	-rm -rf .paasta
 
 yelpy: ## Installs the yelp-internal packages into the default tox environment
-	.tox/py37-linux/bin/pip-custom-platform install -i https://pypi.yelpcorp.com/simple -r yelp_package/extra_requirements_yelp.txt -r ./extra-linux-requirements.txt
```

Comment:
pretty sure that `-r ./extra-linux-requirements.txt` is no longer needed - maybe an artifact of a bygone era when we maybe cared about running on not-linux (our laptops?)